src/application/components/JsonDoc/JsonDoc.js

```diff
@@ -1,7 +1,7 @@
 // @flow
```

Comment:
Fixes the display of `<-- Hover over JSON example for documentation` when there isn't a model.